### ISSUE TITLE
Code : accessing the model elsewhere in the javascript code

### ISSUE THREAD

**Issue by memsis** [pr83/ylc#1]
Hi,

I'm trying to access the model from my controller from outside YellowCode.

I created a function in the controller like this : 
```javascript
var controller = {
	getQuestions: function(model,context){ return model.questions;},
	init: function(model){
		model.questions = [{titre:'je suis une question',type:'ouinon',choix:{'oui':'patate','non':'carotte'}},
				{titre:'je suis une question encore',type:'ouinon',choix:{'oui':'patate','non':'carotte'}}];
	},
	addElement: function(model, context){
		model.questions.push({titre:'',type:'',choix:{}});
	},
	deleteElement: function(model, context){
		model.questions.splice(context.loopStatuses.status.index,1)
	}
};
$(document).ready(function(){
	questions = $("#my").yellowCode(controller).yellowCode("getAdapter");
});
```

But when I access the model by using questions.getQuestions(), I get the initial status of the model and the inputs are resetted to their initial state in the view.

~~I tagged this as "Documentation" since I'm probably not doing it properly, and I think I won't be the only one needing this...~~ _Not revelant anymore_

EDIT : I continued working on this. I found out that if I add a function in the controller that do a console.log(model), then triggers it before calling my getQuestions() function outside the controller, the model get updated. Seems that the 2-way data binding is not working as I though it would... I needed something that would update my model after I changed a value in the DOM...

EDIT #2 : I added a call to v2mProcessElement(my.domView); in the createAdapter() method. I don't know if it's the right thing to do, but it seems to works for what I need.

**Comment by pr83** [comment]
Hello,

Could you please attach your view as well?

Thanks

**Comment by pr83** [comment]
Thanks for flagging it up. This should now be fixed in the latest release (0.99.11) which I created today.
